backend/platform_settings_v2/exceptions.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from rest_framework.exceptions import APIException
+
+
+class InternalServiceError(APIException):
+    status_code = 500
+    default_detail = "Internal error occurred while platform key operations."
+
+
+class UserForbidden(APIException):
+    status_code = 403
+    default_detail = (
+        "User is forbidden from performing this action. Please contact admin"
```

Comment:
```suggestion
        "User is forbidden from performing this action. Please contact admin."
```
NIT: Let's add periods to exception messages shown to the user